core/templates/dev/head/pages/collection-player-page/collection-player-page.directive.html

```diff
@@ -47,12 +47,18 @@ <h2 ng-if="$ctrl.collectionPlaythrough.hasFinishedCollection()" class="oppia-pag
       <a ng-repeat="node in $ctrl.collection.getCollectionNodes()"
          ng-href="<[$ctrl.getExplorationUrl(node.getExplorationId())]>"
          ng-style="{position: 'absolute', left: '<[$ctrl.pathIconParameters[$index].left]>', top: '<[$ctrl.pathIconParameters[$index].top]>'}">
+
+        <div ng-if="[$index]%2===0" ng-style="{width:'160',position:'absolute',color:'#006553','font-size':'20px',bottom:'85%'}">{{node._explorationSummaryObject.title}}</div>
+        <div ng-if="[$index+1]%2===0&&[$index+1]%4!==0" ng-style="{width:'160',position:'absolute',color:'#006553',bottom:'85%','font-size':'20px', left:'20px'}">{{node._explorationSummaryObject.title}}</div>
+        <div ng-if="[$index+1]%4===0" ng-style="{width:'160px',position:'absolute',color:'#006553','font-size':'20px',bottom:'85%',left:'-35px'}">{{node._explorationSummaryObject.title}}</div>
```

Comment:
Try to do all the ``` ng-if ``` functionality, in a function. Don't do all this operation in HTML. Do in Js file.
Like ``` ng-if = function( ) ``` .